packages/ui/Spinner/Spinner.js

```diff
@@ -148,28 +156,31 @@ const Spinner = kind({
 			case 'screen': {
 				return (
 					<FloatingLayer noAutoDismiss open scrimType={scrimType}>
-						<Component {...rest} />
+						<Component ref={componentRef} {...rest} />
 					</FloatingLayer>
 				);
 			}
 			case 'container': {
 				return (
 					<div className={spinnerContainerClassName}>
 						<div className={scrimClassName} />
-						<Component {...rest} />
+						<Component ref={componentRef} {...rest} />
 					</div>
 				);
 			}
 			default: {
 				return (
-					<Component {...rest} />
+					<Component ref={componentRef} {...rest} />
 				);
 			}
 		}
 	}
 });
 
+const Spinner = ForwardRef({prop: 'componentRef'}, SpinnerBase);
```

Comment:
For nearly all these components, doc blocks were not added to reflect the new structure of the components (A base + a HOC wrapped version).  Also, the decorators were not updated to add a `@mixes` section for `ForwardRef`.  For consistency with the docs, we should probably update all these instances _or_ do not bother to export the Base version.